resources/pull_requests.md

```diff
@@ -32,13 +32,28 @@ your context to review your PR
 We already have some Pull Requests templates according to the repository, feel free to
 suggest enhancements for them.
 
-### Metadata
+### Labels
+
+Labels are a great way to organize different types of PR
 
 * Always use labels to allow filtering and better usability in listing
-* As we're a small team, there is no need to indicate `Reviewers`, as all of us probably will
 do code review
+* Select a label that best describes the purpose of PR
+
+### Reviewers
+
+* You can select `Sourcelevel` team/organization as `Reviewers` to notify everybody, so them anyone reviews your PR.
```

Comment:
WDYT:

```suggestion
* You can select the whole team you're working with or people individually. As we use [Pull Reminders](https://pullreminders.com/), it will take care about notifying everyone.
```

Reply:
Fixed in 591a85c.